<launch>

  <!-- TELEOP -->
<group ns="hanse">
  <node name="joy_node" pkg="joy" type="joy_node">
    <param name="dev" value="/dev/input/js0" />
  </node>

  <node name="teleop_joy" pkg="hanse_gamepad" type="teleop_joy" output="screen">
    <rosparam file="$(find hanse_gamepad)/config_xbox.yaml"/>
  </node>
</group>

</launch>
